pkg/grpc/template_config.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/DIMO-Network/vehicle-signal-decoding/pkg/grpc";
+
+package grpc;
+
+message Template {
+  string id  = 1;
+  repeated string pids = 2;
+  string dbc = 3;
+  repeated string template_vehicles = 4;
+}
+message PID {
+  string id = 1;
+}
+
+message GetTemplateListRequest {
+  string powertrain = 1;
+  string protocol = 2;
+
+}
+message GetTemplateListResponse {
+  repeated Template templates = 1;
```

Comment:
I think the object we use for listing should be smaller and different than what we do for viewing/updating/creating. The reason is that if you picture the list of templates, these would just have the properties necessary for listing them in a table, but the properties you see when viewing the template (eg. clicking on the template) are going to be a lot more expanded.

Reply:
Added TemplateSummary to be redacted listing, also added name field